packages/core-ethereum/crates/core-ethereum-indexer/src/handlers.rs

```diff
@@ -360,6 +360,13 @@ where
 
                 let value: U256 = u256::from_be_bytes(transfered.value.into()).into();
 
+                utils_log::debug!(
+                    "on_token_transfer_event - address_to_monitor: {:?} - to: {:?} - from: {:?}",
```

Comment:
You can use the literal directly in the format string as such: `"format text with value of {my_variable}"`

Reply:
I prefer this approach actually.